src/main/java/com/stripe/net/Webhook.java

```diff
@@ -47,10 +48,34 @@ public static Event constructEvent(String payload, String sigHeader, String secr
   public static Event constructEvent(
       String payload, String sigHeader, String secret, long tolerance)
       throws SignatureVerificationException {
-    Event event =
-        StripeObject.deserializeStripeObject(
-            payload, Event.class, ApiResource.getGlobalResponseGetter());
-    Signature.verifyHeader(payload, sigHeader, secret, tolerance);
+    return constructInternalEvent(payload, sigHeader, secret, tolerance, null);
```

Comment:
Why not call the new `constructEvent` overload here to avoid an extra `constructInternalEvent` method?

Reply:
done